Sources/PointFree/Blog/BlogAtomFeed.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Html
+import HttpPipeline
+import MediaType
+import Prelude
+
+let blogAtomFeedResponse =
```

Comment:
jeez it's easy to create an atom feed!

Reply:
but can you spot check this to make sure I got everything right?

Reply:
Yup this all looks right!